jobs/haproxy/spec

```diff
@@ -190,6 +190,9 @@ properties:
   ha_proxy.enable_health_check_http:
     description: "Optionally enable http health-check on `haproxy_ip:8080/health`. It shows `200 OK` if >0 backend servers are up. If used with ext_crt_list_timeout you should make sure that the deployment canary_watch_time and update_watch_time are configured to wait at least the number of seconds defined by ext_crt_list_timeout."
     default: false
+  ha_proxy.enable_monit_health_check_http:
+    description: "Optionally enable http health-check on `haproxy_ip:8080/health` using monit."
```

Comment:
This is misleading. It actually checks (using monit) that the http health endpoint above works as expected. It does not enable another http health endpoint.

Reply:
It should probably also mention that to retain the existing behavior, both flags should have the same value.

Reply:
Hi @domdom82  the issue we have is that when we are starting up the infrastructure the monit test will fail because `HAProxy` won't report anything if the gorouter is not up yet.